packages/documentation/src/pages/forms/available-features-and-usage-guidelines.mdx

```diff
@@ -429,6 +429,7 @@ To configure this feature, place a `preSubmitInfo` object in the `formConfig`. T
 * `field`: The name of the form field for the required checkbox. This field has the value `true` in the submitted form data.
 * `label`: A text string or React element that labels the checkbox.
 * `error`: A text string or React element displayed as an error message if the user attempts to submit the form without checking the checkbox.
+* `CustomComponent`: A React element that will replace the default [`PreSubmitSection`](https://github.com/department-of-veterans-affairs/vets-website/blob/f6a5a742b870869550ab2f5af72428353a5a89da/src/platform/forms-system/src/js/components/PreSubmitSection.jsx#L5) component.
```

Comment:
If you want to follow suit with `CustomComponent`, it looks like the `notice` and `required` options all have "(Optional)" in their descriptions.

Reply:
Will make a prop table